packages/next/src/build/utils.ts

```diff
@@ -1365,7 +1379,17 @@ export async function buildAppStaticPaths({
   requestHeaders: IncrementalCache['requestHeaders']
   nextConfigOutput: 'standalone' | 'export' | undefined
   ComponentMod: AppPageModule
+  isRoutePPREnabled: boolean | undefined
+  isAppPPRFallbacksEnabled: boolean | undefined
 }): Promise<PartialStaticPathsResult> {
+  // If this page isn't dynamic, then we don't need to do anything.
```

Comment:
why do we call `buildAppStaticPaths` on a non-dynamic route? This seems to conflict with the callsite [here](https://github.com/vercel/next.js/blob/feat/pfpr/packages/next/src/build/utils.ts#L1748-L1749) so maybe we can just fix it there instead?